src/disco/containers/DiscoPane.js

```diff
@@ -1,15 +1,26 @@
-import React from 'react';
+import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { asyncConnect } from 'redux-async-connect';
+import { camelCaseProps } from 'core/utils';
 
-import { gettext as _ } from 'core/utils';
+import Addon from '../components/Addon';
+import fakeData from 'disco/fakeData';
```

Comment:
It seems a bit surprising to me that this is set as the default but I assume it's just temporary.